org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeNetworkStateSerializerImpl.java

```diff
@@ -180,4 +180,16 @@ public void deserialize(final ZigBeeNetworkManager networkState) {
         logger.debug("Loading ZigBee network state: Done.");
     }
 
+    public synchronized void delete() {
+        final File file = new File(networkStateFilePath + "/" + networkStateFileName + networkId + ".xml");
```

Comment:
What do you think about extracting this file construction into a helper method? (As, with your change, it is now used at three places in the class.)

Reply:
Good idea.